src/content.js

```diff
@@ -71,6 +72,20 @@ function hideEmptyMeta() {
 	}
 }
 
+// Hide other users starring/forking your repos
+async function hideOwnStars() {
+	const {hideStarsOwnRepos} = await options.getAll();
+
+	if (hideStarsOwnRepos) {
+		const username = getUsername();
+		observeEl('#dashboard .news', () => {
+			$('#dashboard .news .watch_started, #dashboard .news .fork')
+				.has(`a[href^="/${username}"]`)
+				.css('display', 'none');
+		});
+	}
+}
```

Comment:
Move it to its own file? Would be nice to eventually reduce `content.js` to just be a router/delegator.

Reply:
That was the idea on my first refactor, but I'm afraid that we'd have too many 5-line files on the `libs` folder. Surely a lot of the code we have now in `content` is long enough to be in their own file.

Reply:
> but I'm afraid that we'd have too many 5-line files on the libs folder

I don't see that as a problem.

---

I also think we should create a separate directory for the features. Now they're mixed with utils, icons, domify, etc.

Reply:
I might do that after we close a few PRs